crates/sui-transactional-test-runner/src/args.rs

```diff
@@ -48,6 +42,8 @@ pub struct SuiPublishArgs {
 pub struct SuiInitArgs {
     #[clap(long = "accounts", multiple_values(true), multiple_occurrences(false))]
     pub accounts: Option<Vec<String>>,
+    #[clap(long = "protocol_version")]
+    pub protocol_version: Option<u64>,
```

Comment:
I love it, thanks for doing this! Was on someones radar, maybe @tzakian or @rvantonder?